DfE.FindInformationAcademiesTrusts/Program.cs

```diff
@@ -258,6 +264,27 @@ private static void AddAuthenticationServices(WebApplicationBuilder builder)
             });
     }
 
+    private static void AddApiHttpClient(WebApplicationBuilder builder)
+    {
+        builder.Services.AddHttpClient("DfE.FindInformationAcademiesTrusts.API", httpClient =>
+        {
+            var options = new FindInformationAcademiesTrustsApiOptions();
```

Comment:
This will create an empty options object which won't be bound to our options.
I think we can do something like:
```
(serviceProvider, httpClient) =>
    {
        var options = serviceProvider.GetRequiredService<IOptions<FindInformationAcademiesTrustsApiOptions>>().Value;
        httpClient.BaseAddress = new Uri(options. BaseUri);
        httpClient.DefaultRequestHeaders.Add("Accept", "application/json");
    }
```

Reply:
Still having trouble getting options working, it's muddling my head a bit. One to pair on :)